lib/datapackage.py

```diff
@@ -27,30 +27,30 @@ def create_json():
         if script_exists:
             print("Dataset already available. Check the list or try a different shortname")
 
-    contents['title'] = input("Title/Name: ")
-    contents['description'] = input("Description: ")
-    contents['citation'] = input("Citation: ")
-    contents['homepage'] = input("Site/Homepage of dataset: ")
+    contents['title'] = input("Title/Name: ").strip()
+    contents['description'] = input("Description: ").strip()
+    contents['citation'] = input("Citation: ").strip()
+    contents['homepage'] = input("Site/Homepage of dataset: ").strip()
     tags = input("Tags (separated by ';'): ").split(';')
     contents['keywords'] = [tag.strip() for tag in tags if tag.strip() != ""]
     contents['resources'] = []
```

Comment:
I was thinking of writing an input function that gets an argument for the text prompt that is issued, and returns user input only if it is not "" after stripping.